.github/workflows/crypto-primes.yml

```diff
@@ -138,3 +138,16 @@ jobs:
           toolchain: 1.73.0
           profile: minimal
       - run: cargo build --all-features --benches
+
+  cargo-careful:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          profile: minimal
+      - uses: baptiste0928/cargo-install@v3
+        with:
+          crate: "cargo-careful"
+      - run: cargo +nightly careful setup && cargo +nightly careful test
```

Comment:
We probably want `tests-exhaustive` here, or does it make it too slow?

Reply:
No, that should be fine. It runs in ~20 seconds on my machine.